config/production.yaml.example

```diff
@@ -53,6 +53,8 @@ redis:
 
 # SMTP server to send emails
 smtp:
+  transport: smtp
```

Comment:
Please add these keys in https://github.com/Chocobozzz/PeerTube/blob/develop/config/default.yaml too